buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -108,6 +108,8 @@ SPECS = ['ppc64_aix'      : CURRENT_RELEASES,
 SHORT_NAMES = ['all' : ['ppc64le_linux','ppc64le_linux_xl','s390x_linux','s390x_linux_xl','x86-64_linux','x86-64_linux_xl','ppc64_aix','x86-64_windows','x86-32_windows','x86-64_mac'],
             'aix' : ['ppc64_aix'],
             'zlinux' : ['s390x_linux'],
+            'zlinuzcm': ['s390x_linux_cm'],
+            'zlinuzcmake': ['s390x_linux_cm'],
```

Comment:
Need to add to the SPECS variable above as well.
`'s390x_linux_cm' : ['8', '11'],`